### DISCUSSION TITLE
Windows: `clipboard-win` doesn't make sense for helix editor in SSH session.

### DISCUSSION THREAD

**Discussion by noktoborus** [helix-editor/helix#10280] [General]
### Summary

I propose to respect 'SSH_CLIENT' environment variable, e.g. use 'termcode' or another clipboard provider under windows if variable is set.

### Reproduction Steps

Client computer: 

I tried this:
1. login to windows host using ssh
2. `hx`
3. type worlds
4. select all
5. type :clipboard-yank
6. open notepad
7. try to paste
 
I expected this to happen:

Copyied worlds appears in notepad

Instead, this happened:

nothing

Similar steps if I use helix editor in wsl environment (builded for linux), the text is copied to the clipboard.

### Helix log

Windows host through ssh:
```
> hx --health
...
Clipboard provider: clipboard-win
System clipboard provider: clipboard-win
...
```

WSL-host works both if xclip is installed or not installed (proverd: `xclip` or `termcode`)

### Platform

Linux / Windows / Android

### Terminal Emulator

termux

### Installation Method

cargo install

### Helix Version

helix 23.10 (a1272bdb)

**Comment by kirawi** [comment]
Duplicate of https://github.com/helix-editor/helix/issues/8826

**Comment by kirawi** [comment]
Actually, is `SSH_CLIENT` standard?

**Comment by kirawi** [comment]
I feel like this would be solved by the linked issue but I'll leave this to the discretion of the maintainers if they believe there is a way to support this without requiring configuration.
